app/src/main/java/org/p2p/wallet/swap/jupiter/repository/model/SwapToken.kt

```diff
@@ -0,0 +1,14 @@
+package org.p2p.wallet.swap.jupiter.repository.model
+
+import org.p2p.wallet.utils.Base58String
+
+data class SwapToken(
+    val address: Base58String,
+    val chainId: Int,
+    val decimals: Int,
+    val extensions: SwapTokenExtensions,
+    val logoUri: String,
+    val name: String,
+    val symbol: String,
+    val tags: List<String>
```

Comment:
Are you sure we don't have `SwapToken` anymore in the project?
Seems to me that this is a conflict name. 

Let's name it `JupiterSwapToken` please